### DISCUSSION TITLE
ffish.js/ffish.wasm engine commands

### DISCUSSION THREAD

**Discussion by bmacho** [fairy-stockfish/Fairy-Stockfish#312] [Q&A] [answered]
Hey!

I was able to create a board, and make some moves on it, but how can I make it to evaluate a position or play on its own? 

Thank you

**Comment by ianfab** [comment]
Hi. The libraries ffishjs and pyffish only expose the basic chess (variant) rule logic, like move list and FEN generation. If you want to query the engine's search or evaluation, you need to use the native engine or the [WASM version](https://github.com/ianfab/stockfish.wasm).

**Reply by bmacho** [reply]
Thank you! 

Can you provide a working example / is there one available?

**Reply by ianfab** [reply]
No, I do not have one.

**Comment by bmacho** [comment, marked as answer]
There is a working demo here https://fairy-stockfish-nnue-wasm.vercel.app/
